launcher/src/main/java/org/apache/spark/launcher/Main.java

```diff
@@ -101,6 +99,23 @@ public static void main(String[] argsArray) throws Exception {
     }
   }
 
+  /**
+   * Prepare spark commands with the appropriate command builder.
+   * If printLaunchCommand is set then the commands will be printed to the stderr.
+   */
+  private static List<String> buildCommand(
+    AbstractCommandBuilder builder,
```

Comment:
The style is to double indent parameter lists... when in doubt, look at the rest of the code, not the style guide.